src/main/java/jeresources/json/ProfilingAdapter.java

```diff
@@ -102,7 +103,12 @@ public static void write(final Map<Integer, DimensionData> allDimensionData) {
                         writer.endArray();
                     }
 
-                    writer.name("dim").value(DimensionManager.getProvider(dim).getDimensionType().getName());
+                    String name = DimensionManager.getProvider(dim).getDimensionType().getName();
+                	if (name == TranslationHelper.tryDimensionTranslate(name)) {
```

Comment:
Same here as down lower, let's always add the dimension id. In a format like `Dim X: Name`